nautobot/extras/templates/extras/dynamicgroup_edit.html

```diff
@@ -112,5 +114,36 @@
         formCssClass: 'dynamic-formset-{{ children.prefix }}',
         added: jsify_form
     });
+
+    // Function that takes a Dynamic Group children form and recomputes the weights.
+    function update_weights(children_form){
+        let w = 10;
+        $(children_form).find('.formset_row-{{ children.prefix }}').each((i, el) => {
+            let contains_value = $(el).find("select[name$='-group'], select[name$='-operator']").map((idx, sel) => {
+                return ($(sel).val().length > 0)
+            }).toArray().some(x => x === true)
```

Comment:
I can grok that this appears to be arriving at the terminal value of `.some()` (jQuery's analog to Python's `any()`), but would you mind adding a brief comment to explain the intent of this block?